src/openzaak/components/besluiten/tests/models/test_bio_block_change.py

```diff
@@ -44,3 +48,45 @@ def test_bulk_create(self):
         self.assertRaises(
             QueryBlocked, BesluitInformatieObject.objects.bulk_create, [bio]
         )
+
+
+@override_settings(CMIS_ENABLED=True)
```

Comment:
Please move this class to a new file `test_bio_block_change_cmis.py`

Reply:
Addressed all the general test-related comments mentioned at the beginning.